pandera/checks.py

```diff
@@ -1022,3 +1021,43 @@ def _str_length(series: pd.Series) -> pd.Series:
                 f"str_length({min_value}, {max_value})",
             ),
         )
+
+    @classmethod
+    @st.register_check_strategy(st.isin_strategy)
+    @register_check_statistics(["allowed_values"])
+    def mustbein(cls, allowed_values: Iterable, strip_white_space_series: bool = False, **kwargs) -> "Check":
```

Comment:
can we rename this to `unique_values_eq`? `mustbein` is effectively the same as `isin`, it doesn't really communicate the fact that the column must contain all the values specified in the supplied list/set.

Similarly, we can rename `allowed_values` to just `values`, since `allowed_values` implies that column values must be in the specified set, but that not all values in the set are present in the column.